tshirt page changes for review

Sulaymon333 opened the pull request:
<!--- Provide a general summary of the PR in the Title above -->


<!--- Insert the PR's # for the deploy preview's URL -->
[Deploy Preview](https://deploy-preview-5364--keen-mayer-a86c8b.netlify.app/foundation/tshirt-sizing)

#### What does this PR do?
tshirt page changes for review by designers

#### Where should the reviewer start?

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### How should this be manually tested?

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Should this PR be mentioned in Design System updates?

#### Is this change backwards compatible or is it a breaking change?

britt6612 commented:
This image as well is a little big
<img width="1309" height="804" alt="Screenshot 2025-10-02 at 11 33 00 PM" src="https://github.com/user-attachments/assets/57ae2fed-35cb-41ec-94f4-99341c550deb" />

britt6612 commented on aries-site/src/pages/foundation/tshirt-sizing.mdx:
```diff
@@ -28,11 +30,62 @@ Using pixel values, percentages, or other sizing methods to create layouts can r
+**Key highlights of the Dual Doubling scale:**
+
+- **Introduces in-between steps** which address gaps in the current scale, making desired design layouts easier to execute.
+- **Has parity with existing scale** which means that all designs may be recreated with the new scale without any layout shifts except for example container `xlarge` or 1152px doesn't have an equivalent size on the new scale. We recommend using `xxlarge` or 1024px instead on the new scale.
```
```suggestion
- **Has parity with existing scale** which means that all designs may be recreated with the new scale without any layout shifts. The only exception is container `xlarge` (1152px), which does not have a direct equivalent. We recommend using `xxlarge` (1024px) instead.
```

britt6612 commented on aries-site/src/pages/foundation/tshirt-sizing.mdx:
```diff
@@ -28,11 +30,62 @@ Using pixel values, percentages, or other sizing methods to create layouts can r
+- **Introduces in-between steps** which address gaps in the current scale, making desired design layouts easier to execute.
+- **Has parity with existing scale** which means that all designs may be recreated with the new scale without any layout shifts except for example container `xlarge` or 1152px doesn't have an equivalent size on the new scale. We recommend using `xxlarge` or 1024px instead on the new scale.
+- **New `medium` = v6 and below `medium`**; however, rest of t-shirt sizes will need to be mapped to achieve design parity.
+- **Every t-shirt size may be composed** by smaller t-shirt sizes and/or divided into halves, thirds, quarters, ... 12ths, etc. (e.g. `small` + `small` = `large`, or `xsmall` + `xsmall` + `xsmall` = `large`, or even `medium` + `xsmall` = `large`).
```
**T-shirt sizes are modular:** ....

britt6612 commented on aries-site/src/pages/foundation/tshirt-sizing.mdx:
```diff
@@ -53,83 +106,95 @@ One of the main ways t-shirt sizing is used is to control the width/height of a
 The [Grommet site](https://v2.grommet.io/) provides documentation on the props available to any given component. If you're unsure if a component supports `width`, `height`, or `size`, that would be the right place to check.
 
-Given the code snippet from the previous section and the explanation on t-shirt sizing for dimensions, if you were to implement the following code, you would have a Box with width of "384px" and height of "192px".
+Given the [Container sizes comparison](#container-sizes-comparison-width-height-size-columns-rows) table, if you were to implement the following code, you would have a first Box with width of "384px" and height of "192px", second Box with width of "384px" and height of "384px" and a third Box with width of "384px" and height of "768px"
```
```suggestion
Given the [Container sizes comparison](#container-sizes-comparison-width-height-size-columns-rows) table the following code would render 3 boxes

- The first Box has a width of 384px and height of 192px.
- The second Box has a width of 384px and height of 384px.
- The third Box has a width of 384px and height of 768px.
```


not sure about this suggestion or how this helps user open to other thoughts

britt6612 commented on aries-site/src/pages/foundation/tshirt-sizing.mdx:
````diff
@@ -53,83 +106,95 @@ One of the main ways t-shirt sizing is used is to control the width/height of a
-
-}
-```
+Pad, gap, and margin are controlled by `edgeSize` which now includes the expanded scale shown in the spacing sizes section above.
````
```suggestion
`pad`, `gap`, and `margin` are controlled by `edgeSize` which now includes the expanded scale shown in the spacing sizes section above.
```

Sulaymon333 commented:
The large image is due to the branch out of sync with changes from `site-theme-v7` which was not previosly merged to master. I have updated that to have the changes now since `site-theme-v7` is also merged into master already and it is now fixed.

<img width="885" height="1025" alt="image" src="https://github.com/user-attachments/assets/bf42add8-de6e-46df-97f1-7812c26fea13" />

<img width="659" height="270" alt="image" src="https://github.com/user-attachments/assets/7fe7aecb-4fae-4f97-9ee8-03a4798a2b5f" />

<img width="689" height="276" alt="image" src="https://github.com/user-attachments/assets/e0d9ac9d-569f-4701-896d-e72ff9b594e9" />

<img width="875" height="525" alt="image" src="https://github.com/user-attachments/assets/1b8b7fae-22ed-4b64-9770-cba387d8902e" />

jcfilben commented on aries-site/src/pages/foundation/tshirt-sizing.mdx:
```diff
@@ -2,37 +2,120 @@ import { Box, Image } from 'grommet';
+
+- Named size tokens (e.g. `small`, `medium`) replace raw pixel values.
+- The system governs both component dimensions and spacing (padding, gap, margin, border, radius).
+- The scale derives from two base units: 24px (original) and 16px (supplemental in v7).
```
We jump in to talking about v7 and v6. Since we have a lot of different package offering s let's make it clear that we are referring to grommet-theme-hpe here.

Let's also try to structure the information in such a way that it will hold up long term. For example if we are on v9 or v10 of the theme, this page should still make sense to users. The way that this reads currently is that the supplemental base unit only applys to v7

jcfilben commented on aries-site/src/pages/foundation/tshirt-sizing.mdx:
```diff
@@ -47,118 +130,124 @@ xxlarge: `${baseSpacing * 64}px`, // 1536px
+
+### Composition visualization
+
+The beauty of the Dual Doubling scale is its mathematical precision, allowing sizes to compose and decompose in predictable ways. The visualization below demonstrates how different t-shirt sizes relate to each other:
```
In some places we refer to the 'v7 scale' in others we say 'the Dual Doubling scale', thinking we should align to a consistent language.

I'm wondering if we should refer to the v6 and earlier scale as the 'legacy' t-shirt scale and v7 and above is just the t-shirt size scale.
